recipe/conda_build_config.yaml

```diff
@@ -2,26 +2,34 @@ macos_machine:
   - x86_64-apple-darwin13.4.0  # [not (osx and arm64)]
   - x86_64-apple-darwin13.4.0  # [not (osx and arm64)]
   - x86_64-apple-darwin13.4.0
+  - x86_64-apple-darwin13.4.0
+  - arm64-apple-darwin20.0.0
   - arm64-apple-darwin20.0.0
   - arm64-apple-darwin20.0.0
 cross_target_platform:
   - osx-64  # [not (osx and arm64)]
   - osx-64  # [not (osx and arm64)]
   - osx-64
+  - osx-64
+  - osx-arm64
   - osx-arm64
   - osx-arm64
 gfortran_version:
   - 9.5.0   # [not (osx and arm64)]
   - 10.4.0  # [not (osx and arm64)]
   - 11.3.0
+  - 12.2.0
   - 10.3.0
   - 11.3.0
+  - 12.2.0
 libgfortran_version:
   - 5.0.0   # [not (osx and arm64)]
   - 5.0.0   # [not (osx and arm64)]
   - 5.0.0
   - 5.0.0
   - 5.0.0
+  - 5.0.0
+  - 5.0.0
```

Comment:
As discussed elsewhere idk if these versions are correct. I assume @isuruf knows. It'd be even cooler to know how to find this information myself!

Reply:
I was basing this on the `NEWS` in the source code that has no comments about `libfortran` version changing since gcc 8.  I was also going to look at the build artifacts and make sure that the dylib with version 5 was built.  (Assuming it compiles successfully.)